packages/markers/src/browser/marker-tree.ts

```diff
@@ -126,7 +127,7 @@ export abstract class MarkerTree<T extends object> extends TreeImpl {
     }
 }
 
-export interface MarkerNode extends UriSelection, SelectableTreeNode {
+export interface MarkerNode extends UriSelection, SelectableTreeNode, ProblemSelection {
```

Comment:
you probably should update `MarkerNode.is` to use `ProblemSelection.is` instead of `'marker' in node`